package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "test": "jest",
-    "test:coverage": "jest --coverage"
+    "test:coverage": "jest --coverage",
+    "lint": "eslint 'lib/**/*.js' --fix"
```

Comment:
```suggestion
    "lint": "eslint 'lib/**/*.js'",
    "lint:fix": "eslint 'lib/**/*.js --fix'"
```

Reply:
Yes, in a few hours I will reconnect and I will